instructor/ui/sketch_tool.js

```diff
@@ -129,7 +131,8 @@ function draw_line(x1 ,y1 ,x2 ,y2, color){
 
 }
 
-function nameXAxis(condition){
+function nameXAxis(condition, this_paper){
+    paper = this_paper;
```

Comment:
`paper` is a global variable that keeps reference to the currently active PaperScope. So here I am trying to set the PaperScope I want to work with. I found a better way to do this: `this_paper.activate()`.